l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py

```diff
@@ -42,26 +42,27 @@ def get_transaction_line_count_from_xml(auditfile):
 # (esp. product.template).
 @tagged("-at_install", "post_install")
 class TestXafAuditfileExport(TransactionCase):
```

Comment:
The tests are failing because in setUpClass cls.env is indeed not defined. Either keep the test as is or change TransactionCase into SavepointCase. Preferably keep test as is, as in later Odoo released SavepointCase will be deprecated (actually TransactionCase will work then as SavepointCase now).

Reply:
In 16.0 cls.env would be defined in setUpClass...

Reply:
Yes I see, the cls.env is defined if I set the SavepointCase.
Making use of the setUpClass() was just an attempt to make tests pass. Seems not working, so I'm reverting it and keep the original setUp().
The tests failure is because the SQL code present in the module seems not returning the expected data. It was always working fine in the past but now not anymore. Do you have any idea of why is that happening?